perma_web/perma/settings/deployments/settings_common.py

```diff
@@ -157,7 +157,6 @@ def _pywb_static_dir_location():
 
```

Comment:
(We'll want to make sure that variable is set to v1 on all the various tiers too.)